open-media-match/src/OpenMediaMatch/__init__.py

```diff
@@ -1 +1,30 @@
 # Copyright (c) Meta Platforms, Inc. and affiliates.
+
+import flask
+
+
+def create_app():
+    """
+    Create and configure the Flask app
+    """
+    app = flask.Flask(__name__, instance_relative_config=True)
+
+    @app.route("/")
+    def ping():
```

Comment:
is this ping or status?

Reply:
You're right, those aren't the same thing. Will make the distinction